mne/preprocessing/ica.py

```diff
@@ -132,11 +132,12 @@ class ICA(ContainsMixin):
         >> ica.fit(raw)
         >> raw.info['projs'] = projs
 
-    .. note:: Methods implemented are FastICA (default), Infomax and
-              Extended-Infomax. Infomax can be quite sensitive to differences
-              in floating point arithmetic due to exponential non-linearity.
-              Extended-Infomax seems to be more stable in this respect
-              enhancing reproducibility and stability of results.
+    .. note:: Methods implemented are FastICA (default), Infomax,
+              Extended-Infomax, and Picard. Infomax can be quite sensitive to
+              differences in floating point arithmetic due to exponential
+              non-linearity. Extended-Infomax seems to be more stable in this
+              respect enhancing reproducibility and stability of results.
+              Picard tends to be faster than the other algorithms.
```

Comment:
Maybe we should add the references here.

Reply:
Done :)